### PR TITLE
[IMP][ADD] account_credit_control: add migration script from account_…

### PR DESCRIPTION
**Pull request by cubells**
…followup

- Add scripts to migrate data if account_followup is installed.

cc @Tecnativa

### CODE REVIEW

File: account_credit_control/__openerp__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': 'Account Credit Control',
-    'version': '9.0.0.1.0',
+    'version': '9.0.0.2.0',
```
Comment (pedrobaeza) [inline account_credit_control/__openerp__.py]: Version is incorrect (well, it was also incorrect on first iteration), but here you don't get anything changing it.

File: account_credit_control/migrations/9.0.1.0.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def migrate_followup_data(env):
+    env.cr.execute(
```
Comment (pedrobaeza) [inline account_credit_control/migrations/9.0.1.0.0/post-migration.py:9]: These scripts are not going to work unless you include in OpenUpgrade a renaming from `account_followup` to `account_credit_control`.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_credit_control/migrations/9.0.1.0.0/post-migration.py (hunk 2 above)
Comment (cubells) [reply account_credit_control/migrations/9.0.1.0.0/post-migration.py:9]: Related: https://github.com/OCA/OpenUpgrade/pull/1035

**Comment by pedrobaeza** [comment]
But the module version is still incorrect (it puts `9.0.0.1.0` instead of `9.0.1.0.0`)...

**Comment by cubells** [comment]
@pedrobaeza 

account_credit_module was merged with an incorrect version number. :)

**Review by MiquelRForgeFlow — APPROVED** [review]
code review :+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**